.cursor/mcp.json

```diff
@@ -0,0 +1,7 @@
+{
+  "mcpServers": {
+    "nx-mcp": {
+      "url": "http://localhost:9376/mcp"
+    }
```

Comment:
suggestion/question: Can we also add the Angular MCP, or does the Nx one cover it with the Angular plugin?

Reference: https://angular.dev/ai/mcp

Also, it would be nice to make a copy of this file with the `servers` key for VS Code. That would cover the prominent editors except JetBrains, which does not support a code-based way to configure MCP servers.